custom/sensors/CANUltrasonicDistanceSensor.java

```diff
@@ -17,20 +15,20 @@
 	public CANUltrasonicDistanceSensor(String name, int id) {
 		super(name, id);
 	}
-	
+
 	@Override
 	public double getDistance() {
 		try {
-			return getDistance();
+			return getDistanceSafely();
 		}
 		catch (Exception e) {
 			LogKitten.ex(e);
 			return 0;
 		}
 	}
-	
+
 	@Override
 	public double getDistanceSafely() throws InvalidSensorException {
-		return super.read(CANUltrasonicDistanceSensor.CAN_SENSOR_MODE);
+		return super.readSensor()[0];
```

Comment:
why is CAN_SENSOR_MODE no longer a constant? what previously was `super.read(CANUltrasonicDistanceSensor.CAN_SENSOR_MODE);` could become `super.readSensor()[CANUltrasonicDistanceSensor.CAN_SENSOR_MODE];`, no?